### DISCUSSION TITLE
website error

### DISCUSSION THREAD

**Discussion by Mehri06** [community/community#58812] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am getting error while creating my website.

**Comment by the-officialjosh** [comment]
what is the error like trace it that way i can help resolve it better

**Reply by Mehri06** [reply]
404

File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

Read the full documentation for more information about using GitHub Pages.

**Reply by the-officialjosh** [reply]
Is the repo public so I can look at it

**Reply by Mehri06** [reply]
yes

**Reply by Mehri06** [reply]
https://github.com/Mehri06/CiscoPT

**Reply by the-officialjosh** [reply]
It appears that you are missing a crucial element - an `index.html` file. In order for `gh-pages` to function properly, a root folder must be established, and an `index.html` file must be present within it. Have you verified whether this project displays correctly on your local host?

**Reply by Mehri06** [reply]
oh I got it let me make a html file and then I will try it again.

**Reply by the-officialjosh** [reply]
Before proceeding further, make sure to test and verify that the solution works on your local browser. Once you have confirmed the functionality, you can mark the previous reply as the accepted answer and close this issue. By doing so, others experiencing a similar issue can find this solution and benefit from it.
